.gitignore

```diff
@@ -1,2 +1,3 @@
 doc
 pkg
+.DS_Store
```

Comment:
What you think about add this only to your global gitignore since this is a OS file related.

https://robots.thoughtbot.com/global-gitignore